client/my-sites/domains/controller.jsx

```diff
@@ -299,7 +298,7 @@ const jetpackNoDomainsWarning = ( context, next ) => {
 	const state = context.store.getState();
 	const selectedSite = getSelectedSite( state );
 
-	if ( selectedSite && selectedSite.jetpack && ! isATEnabled( selectedSite ) ) {
+	if ( selectedSite && selectedSite.jetpack && ! selectedSite.options.is_automated_transfer ) {
```

Comment:
Can we use the `isSiteAutomatedTransfer` selector here? Or if we prefer not using that selector, let's be consistent and remove it completely.

Reply:
Done in 5c47846b8e. There are quite a few pre-existing places that also look at `options.is_automated_transfer` without abstracting it to any helper.

Reply:
Yeah, same as `site.jetpack` when we have a dedicated selector. 🤷‍♂️